### PR TITLE
[Nexus] some cleanups (see commits)

### PR DESCRIPTION
**Pull request by AlwinEsch**
First thought to do a cleanup like the other addons before the Nexus branch comes, but you already have it, cool.

Therefore, here are the intended changes directly to the Nexus version.

Version increase is not yet included and will first ask which one it should be. 2.9.5 or 20.0.0, what do we do with most of the add-ons in order to have the corresponding Kodi version?

### CODE REVIEW

**Comment by AlwinEsch** [comment]
Can you look to github integrations here and on other addon about installed parts and Jenkins present:
Bildschirmfoto vom 2021-09-17 00-02-00

The Pull Requests are not reported to here and build not triggered automatic: https://jenkins.kodi.tv/job/flubshi/job/pvr.waipu/view/change-requests/
For here it has worked after triggered manual on Jenkins.

**Comment by flubshi** [comment]
In the pvr.waipu Github intergrations Jenkins is not available. If it helps CI to add it, please provide a hint what I have to do.

File: pvr.waipu/resources/language/resource.language.de_de/strings.po [hunk 1]
```diff
@@ -10,6 +10,14 @@ msgstr ""
...
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
+msgctxt "Addon Summary"
+msgid "waipu.tv PVR Client"
+msgstr "waipu.tv PVR Client"
+
+msgctxt "Addon Description"
+msgid "waipu.tv PVR Client. Note: This is not an official waipu plugin and not related to Exaring AG or waipu.tv. It was developed by volunteers."
+msgstr "waipu.tv PVR Client. Hinweis: Dies ist kein offizielles Waipu-Plugin und steht nicht in Verbindung zur Exaring AG oder zu waipu.tv. Es wurde von Freiwilligen entwickelt."
+
```
Comment (flubshi) [inline pvr.waipu/resources/language/resource.language.de_de/strings.po:20]: Where is this used? Or why did you copy it from `addon.xml.in`?

Comment (AlwinEsch) [reply pvr.waipu/resources/language/resource.language.de_de/strings.po:20]: Needed to have available to https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-waipu/ for translation, the https://github.com/flubshi/pvr.waipu/blob/Nexus/.github/workflows/sync-addon-metadata-translations.yml bring them then back to addon.xml.in after change.

Comment (AlwinEsch) [reply pvr.waipu/resources/language/resource.language.de_de/strings.po:20]: Seen this https://github.com/flubshi/pvr.waipu/blob/Nexus/.github/workflows/sync-addon-metadata-translations.yml#L7 must also be fixed, the addon.xml on line should be addon.xml.in.

**Comment by AlwinEsch** [comment]
> In the pvr.waipu Github intergrations Jenkins is not available. If it helps CI to add it, please provide a hint what I have to do.

Think @lrusak know more about.

**Pull request merged**